docs/executor-cypress.md

```diff
@@ -24,15 +24,15 @@ describe('The Home Page', () => {
 })
 ```
 
-## Creating `testkube` test script
+## Creating the TestKube Test Script
 
-Now we need to create TestKube test script from this example (we need to pass repo, branch, path where project exists - in case of mono repo, name and type)
+Create the TestKube test script from this example. The parameters passed are **repository**, **branch** and **the path where project exists**. In the case of a mono repository, the parameters are **name** and **type**.
 
 ```sh
 kubectl testkube scripts create --uri https://github.com/kubeshop/testkube-executor-cypress.git --git-branch main --git-path examples --name kubeshop-cypress --type cypress/project
 ```
```

Comment:
@jfermi @vsukhin 
Where do you think would be the best place to add an example for the private repo ?